Notebooks/OOPS_TMT_Data_Processing_And_QC.Rmd

```diff
@@ -204,25 +230,40 @@ colnames(OOPS_psm.e) <- gsub('Abundance.', 'Input', colnames(OOPS_psm.e))
 # 'expression' data and 'feature' data
 
 
-OOPS_psm.p <- data.frame(Treatment=rep(factor(c('Vehicle', 'Safe', 'Toxic','Vehicle', 'Safe', 'Toxic','Vehicle', 'Safe', 'Toxic','Vehicle', 'Safe', 'Toxic','Vehicle', 'Safe', 'Toxic')), c(1,1,1,1,1,1,1,1,1,1,1,1,1,1,1)), row.names=colnames(OOPS_psm.e))
+OOPS_psm.p <- data.frame(Treatment=factor(rep(c('Vehicle', 'Safe', 'Toxic'), times=5)), row.names=colnames(OOPS_psm.e))
   
 OOPS_psm <- MSnbase::MSnSet(exprs = OOPS_psm.e, fData = OOPS_psm.f, pData=OOPS_psm.p)
 ```
 
+We have some rows in our PSM quantification where all samples are NA. We want to remove these
+```{r}
+table(rowSums(is.na(exprs(OOPS_psm))))
+OOPS_psm <- OOPS_psm[rowSums(is.na(exprs(OOPS_psm))) < ncol(exprs(OOPS_psm))]
+table(rowSums(is.na(exprs(OOPS_psm))))
+```
+
 Plot intensity distributions
 
 ```{r}
 plot_quant(log(OOPS_psm, base=2), method='density')
 ```
 We want to remove low Signal:Noise (S:N) PSMs, since the quantification values will be less accurate and there will be more missing values. We can inspect the relationship between S:N and missing values using the plot_missing_SN function.
 ```{r}
-plot_missing_SN(OOPS_psm, bins = 40)
+plot_missing_SN(OOPS_psm[fData(OOPS_psm)$Average.Reporter.SN<100,], bins = 25)
 ```
-We can also look into this relationship at the tag level using plot_missing_SN_per_sample. In this case, there is no tag which appears to have a high proportion of missing values when signal:noise > 5. If there were, this may warrant further exploration, e.g was one of the sample preparations inadequate such that fewer peptides were labeled?
+We can also look into this relationship at the tag level using plot_missing_SN_per_sample. In this case, although there are tags which have a higher proportion of missing values when signal:noise > 5, this does appear to relate to the experimental condition at all. 
 ```{r}
-plot_missing_SN_per_sample(OOPS_psm, bins = 40)
+# new object with updated sample names so the plot is more informative
```

Comment:
S:N vs missing plot is hard to interpret when sample names are tags, e.g no experimental information. This creates new object with more useful sample names (just for this plot)